x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/constants.ts

```diff
@@ -22,3 +22,25 @@ export const CREATE_AN_ENGINE_BUTTON_LABEL = i18n.translate(
     defaultMessage: 'Create an engine',
   }
 );
+
+export const CREATE_A_META_ENGINE_BUTTON_LABEL = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engines.createAMetaEngineButton.ButtonLabel',
+  {
+    defaultMessage: 'Create a meta engine',
+  }
+);
+
+export const META_ENGINE_EMPTY_PROMPT_TITLE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engines.metaEngines.emptyPrompTitle',
+  {
+    defaultMessage: 'No meta engines yet',
+  }
+);
+
+export const META_ENGINE_EMPTY_PROMPT_DESCRIPTION = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.engines.metaEngines.emptyPromptDescription',
+  {
+    defaultMessage:
+      'Meta engines allow you to combine multiple engines into one, searchable engine.',
```

Comment:
Ah, just realized this is duplicated from https://github.com/elastic/kibana/pull/92127#discussion_r582179543 - I'm aware you've said before that there's a purpose to splitting out (e.g.) titles vs button labels, but in this scenario I feel that doesn't apply as much (they're both `Description` contexts) and we should DRY this out. I personally can't imagine a scenario where a translator would localize this specific differently.

Reply:
@byronhulcher Not sure if you missed this comment - any thoughts here on combining these identical descriptions into a single var?

Reply:
I think it makes sense to keep them separate since they'e different contexts.

Reply:
How are they different contexts? They're both `Description` type paragraph strings (vs being used in a title heading vs a button). Can you give an example of where an i18n localizer would translate this sentence differently in one view vs. another?

Also, another point for combining these: `one searchable` vs `one, searchable` was fixed in another instance of this duplicated string but not this one.

Reply:
Sorry I should have rephrased my previous comment to be less pointed, I'm genuinely just very curious about the i18n scenarios that would require these to be differently ID'd strings. I'd be fine with approving this as-is with the comma removed to match the other var.

```suggestion
      'Meta engines allow you to combine multiple engines into one searchable engine.',
```

Reply:
I think it sounds like you kinda already made up your mind around this, in the future please just make a change request 😃

Reply:
I haven't made up my mind - I don't know if there's something I don't know about i18n, so I asked for a clarification or explanation that would help me understand. Is there something about my messaging that is indicating otherwise / that I'm not interested in hearing/discussing more?

Reply:
Sorry I think my sass came off wrong, happy to add the comma.

Reply:
Sorry, I misread your comment. Apologies for the flip response. I will add the missing comma and hopefully we can continue discussion of shared constants maybe outside this github issue.

Reply:
Definitely looking forward to discussing & learning more! Wherever is fine, either continuing this thread or Slack etc!